packages/calcite-components/src/components/combobox-item/combobox-item.scss

```diff
@@ -1,3 +1,18 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-combobox-item-text-color: Specifies the text color of the component.
```

Comment:
I'm still a bit confused on the naming here. Do we need the `-item` in this var? Because we don't see `-item` in the var `--calcite-comboobx-description-text-color`. Wouldnt this var just be named `--calcite-combobox-text-color`?

Reply:
Earlier I added a token `--calcite-combobox-text-color` in combobox and changed it to `--calcite-combobox-input-text-color`. 

`--calcite-combobox-item-text-color` should be updated to `--calcite-combobox-text-color`.